src/sentry/static/sentry/app/components/assistant/guideAnchor.tsx

```diff
@@ -59,8 +59,8 @@ const GuideAnchor = createReactClass<Props, State>({
   },
 
   componentDidMount() {
-    const {target} = this.props;
-    target && registerAnchor(target);
+    const {target, disabled} = this.props;
+    !disabled ? target && registerAnchor(target) : null;
```

Comment:
@scefali has come across the same problem in https://github.com/getsentry/sentry/pull/24649/files#diff-64f39399f685b89750f3733e99450ea50e72954898e8a68609cbc657e431dce1R213-R222

I think using a wrapper as opposed to checking disabled in componentDidMount is more clear because disabled could change and I would expect the anchor to mount, where here we're checking disabled once.

Reply:
Working on getting my PR merged so you can take advantage of my change :)

Reply:
@mikellykels just merged my PR so you don't need this change

Reply:
@scefali awesome! 👍🏼